papermill/execute.py

```diff
@@ -260,13 +265,12 @@ def build_r_params(parameters):
     """Writes parameters assignment code for R kernels."""
     param_content = "# Parameters\n"
     for var, val in parameters.items():
-        if isinstance(val, string_types):
-            val = '"%s"' % val.replace('"', '\\"')
-        elif val is True:
+        val = _form_escaped_value(val)
+        if val is True:
             val = 'TRUE'
         elif val is False:
             val = 'FALSE'
-        param_content += '%s = %s\n' % (var, val)
+        param_content += '{} = {}\n'.format(var, val)
     return param_content
```

Comment:
Side note, since codecov shows me coverage info inline on GitHub -- this section of code is not covered. May want some basic tests here.